db/fixtures/miq_policy_sets.yml

```diff
@@ -39,6 +39,21 @@
           description: Mark as Non-Compliant
           action_type: default
           options: {}
+      - qualifier: failure
+        failure_sequence: 2
+        failure_synchronous: true
+        MiqEventDefinition:
+          name: containerimage_compliance_check
+          description: Container Image Compliance Check
+          event_type: Default
+          definition:
+          default:
+          enabled:
+        MiqAction:
+          name: container_image_annotate_deny_execution
+          description: Prevent container image from running on OpenShift
+          action_type: default
```

Comment:
I realize the rest of the file is this way but... 

:trollface: because `action_type: Default` would be too consistent with `event_type: Default`.

Can you spend a few minutes to track this down to see if this is fixable in a followup PR or is too hard?  I don't know if anything is querying based on these case sensitive types.